### ISSUE TITLE
Unhandled AttributeError when no config files found

### ISSUE THREAD

**Issue by qubidt** [nicolargo/glances#1569]
#### Description

**update**: description incorrect, see https://github.com/nicolargo/glances/issues/1569#issuecomment-565742311

---

Without any config files defined, [this logger call](https://github.com/nicolargo/glances/blob/e29a153d93a740397d2c7cfa256596240b9bc2b9/glances/plugins/glances_plugin.py#L89-L90) triggers an `AttributeError`.

It looks like this was introduced by a61c91b111f06523e8f265b7fcd63284833ad0dc.

I would write a PR but I'm not sure whether you'd like this resolved by fixing the `logger.debug` call or by providing a `Config` object when there are no config files to read. (I assume the former, though)

Stacktrace and logs included below.

#### Versions

* Glances & psutil (glances -V): `Glances v3.1.3 with psutil v5.6.7`
* Operating System (lsb_release -a): 
```
LSB Version:    1.4
Distributor ID: Arch
Description:    Arch Linux
Release:        rolling
Codename:       n/a
```

#### Logs

```
$ glances -c localhost
Traceback (most recent call last):
  File "/usr/bin/glances", line 11, in <module>
    load_entry_point('Glances==3.1.3', 'console_scripts', 'glances')()
  File "/usr/lib/python3.8/site-packages/glances/__init__.py", line 143, in main
    start(config=config, args=args)
  File "/usr/lib/python3.8/site-packages/glances/__init__.py", line 112, in start
    mode.serve_forever()
  File "/usr/lib/python3.8/site-packages/glances/client.py", line 238, in serve_forever
    if not self.login():
  File "/usr/lib/python3.8/site-packages/glances/client.py", line 162, in login
    if not self._login_glances():
  File "/usr/lib/python3.8/site-packages/glances/client.py", line 130, in _login_glances
    self.stats.set_plugins(json.loads(self.client.getAllPlugins()))
  File "/usr/lib/python3.8/site-packages/glances/stats_client.py", line 59, in set_plugins
    self._plugins[item] = plugin.Plugin(args=self.args)
  File "/usr/lib/python3.8/site-packages/glances/plugins/glances_connections.py", line 60, in __init__
    super(Plugin, self).__init__(args=args,
  File "/usr/lib/python3.8/site-packages/glances/plugins/glances_plugin.py", line 90, in __init__
    config.config_file_paths()))
AttributeError: 'NoneType' object has no attribute 'config_file_paths'
```

```
2019-12-13 10:11:03,817 -- INFO -- Start Glances 3.1.3
2019-12-13 10:11:03,817 -- INFO -- CPython 3.8.0 and psutil 5.6.7 detected
2019-12-13 10:11:03,822 -- INFO -- Search glances.conf file in /home/qubidt/.config/glances/glances.conf
2019-12-13 10:11:03,822 -- INFO -- Search glances.conf file in /etc/glances/glances.conf
2019-12-13 10:11:03,826 -- INFO -- Start GlancesClient mode
2019-12-13 10:11:03,826 -- DEBUG -- Try to connect to http://localhost:61209
2019-12-13 10:11:03,826 -- DEBUG -- Glances started in 0.004254 seconds
2019-12-13 10:11:03,831 -- DEBUG -- Pystache library not found (action scripts won't work)
2019-12-13 10:11:03,831 -- DEBUG -- Load section connections in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,832 -- DEBUG -- Plugin glances_connections.py started in 0.002044 seconds
2019-12-13 10:11:03,832 -- DEBUG -- Load section amps in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,832 -- DEBUG -- AMPs list: []
2019-12-13 10:11:03,832 -- DEBUG -- Plugin glances_amps.py started in 0.000581 seconds
2019-12-13 10:11:03,833 -- DEBUG -- Stats history activated for plugin fs (items: ['percent'])
2019-12-13 10:11:03,833 -- DEBUG -- Load section fs in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,833 -- DEBUG -- Load limit: fs_careful = 50.0
2019-12-13 10:11:03,833 -- DEBUG -- Load limit: fs_warning = 70.0
2019-12-13 10:11:03,833 -- DEBUG -- Load limit: fs_critical = 90.0
2019-12-13 10:11:03,833 -- DEBUG -- Plugin glances_fs.py started in 0.000578 seconds
2019-12-13 10:11:03,833 -- DEBUG -- Stats history activated for plugin network (items: ['rx', 'tx'])
2019-12-13 10:11:03,833 -- DEBUG -- Load section network in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,833 -- DEBUG -- Load limit: network_rx_careful = 50.0
2019-12-13 10:11:03,833 -- DEBUG -- Load limit: network_rx_warning = 70.0
2019-12-13 10:11:03,833 -- DEBUG -- Load limit: network_rx_critical = 90.0
2019-12-13 10:11:03,833 -- DEBUG -- Load limit: network_tx_careful = 50.0
2019-12-13 10:11:03,833 -- DEBUG -- Load limit: network_tx_warning = 70.0
2019-12-13 10:11:03,834 -- DEBUG -- Load limit: network_tx_critical = 90.0
2019-12-13 10:11:03,834 -- DEBUG -- Plugin glances_network.py started in 0.000694 seconds
2019-12-13 10:11:03,834 -- DEBUG -- batinfo library not found. Fallback to psutil.
2019-12-13 10:11:03,835 -- DEBUG -- Load section sensors in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,835 -- DEBUG -- Load limit: sensors_temperature_core_careful = 60.0
2019-12-13 10:11:03,835 -- DEBUG -- Load limit: sensors_temperature_core_warning = 70.0
2019-12-13 10:11:03,835 -- DEBUG -- Load limit: sensors_temperature_core_critical = 80.0
2019-12-13 10:11:03,835 -- DEBUG -- Load limit: sensors_temperature_hdd_careful = 45.0
2019-12-13 10:11:03,835 -- DEBUG -- Load limit: sensors_temperature_hdd_warning = 52.0
2019-12-13 10:11:03,835 -- DEBUG -- Load limit: sensors_temperature_hdd_critical = 60.0
2019-12-13 10:11:03,835 -- DEBUG -- Load limit: sensors_battery_careful = 80.0
2019-12-13 10:11:03,835 -- DEBUG -- Load limit: sensors_battery_warning = 90.0
2019-12-13 10:11:03,835 -- DEBUG -- Load limit: sensors_battery_critical = 95.0
2019-12-13 10:11:03,840 -- DEBUG -- Generic sensor plugin init duration: 0.004506 seconds
2019-12-13 10:11:03,840 -- DEBUG -- Load section hddtemp in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,840 -- DEBUG -- HDDTemp sensor plugin init duration: 9.4e-05 seconds
2019-12-13 10:11:03,840 -- DEBUG -- Load section batpercent in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,840 -- DEBUG -- Battery sensor plugin init duration: 6.5e-05 seconds
2019-12-13 10:11:03,840 -- DEBUG -- Plugin glances_sensors.py started in 0.006399 seconds
2019-12-13 10:11:03,841 -- WARNING -- Missing Python Lib (No module named 'wifi'), Wifi plugin is disabled
2019-12-13 10:11:03,841 -- WARNING -- Wifi lib is not compliant with Python 3, Wifi plugin is disabled
2019-12-13 10:11:03,841 -- DEBUG -- Load section wifi in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,841 -- DEBUG -- Plugin glances_wifi.py started in 0.000723 seconds
2019-12-13 10:11:03,917 -- DEBUG -- Load section ports in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,917 -- DEBUG -- No [ports] section in the configuration file. Cannot load ports list.
2019-12-13 10:11:03,917 -- DEBUG -- No [ports] section in the configuration file. Cannot load ports list.
2019-12-13 10:11:03,917 -- DEBUG -- Plugin glances_ports.py started in 0.076336 seconds
2019-12-13 10:11:03,917 -- WARNING -- Missing Python Lib (No module named 'pynvml'), Nvidia GPU plugin is disabled
2019-12-13 10:11:03,918 -- DEBUG -- Load section gpu in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,918 -- DEBUG -- pynvml could not be initialized.
2019-12-13 10:11:03,918 -- DEBUG -- Plugin glances_gpu.py started in 0.000472 seconds
2019-12-13 10:11:03,918 -- DEBUG -- Load section help in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,918 -- DEBUG -- Plugin glances_help.py started in 0.000284 seconds
2019-12-13 10:11:03,918 -- DEBUG -- Load section system in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,918 -- DEBUG -- Plugin glances_system.py started in 0.000273 seconds
2019-12-13 10:11:03,918 -- DEBUG -- Stats history activated for plugin processcount (items: ['total', 'running', 'sleeping', 'thread'])
2019-12-13 10:11:03,919 -- DEBUG -- Load section processcount in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,919 -- DEBUG -- Plugin glances_processcount.py started in 0.000265 seconds
2019-12-13 10:11:03,919 -- DEBUG -- Load section processlist in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,919 -- DEBUG -- Load limit: processlist_cpu_careful = 50.0
2019-12-13 10:11:03,919 -- DEBUG -- Load limit: processlist_cpu_warning = 70.0
2019-12-13 10:11:03,920 -- DEBUG -- Load limit: processlist_cpu_critical = 90.0
2019-12-13 10:11:03,920 -- DEBUG -- Load limit: processlist_mem_careful = 50.0
2019-12-13 10:11:03,920 -- DEBUG -- Load limit: processlist_mem_warning = 70.0
2019-12-13 10:11:03,920 -- DEBUG -- Load limit: processlist_mem_critical = 90.0
2019-12-13 10:11:03,920 -- DEBUG -- Plugin glances_processlist.py started in 0.001099 seconds
2019-12-13 10:11:03,920 -- DEBUG -- Stats history activated for plugin mem (items: ['percent'])
2019-12-13 10:11:03,920 -- DEBUG -- Load section mem in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,920 -- DEBUG -- Load limit: mem_careful = 50.0
2019-12-13 10:11:03,920 -- DEBUG -- Load limit: mem_warning = 70.0
2019-12-13 10:11:03,920 -- DEBUG -- Load limit: mem_critical = 90.0
2019-12-13 10:11:03,920 -- DEBUG -- Plugin glances_mem.py started in 0.000408 seconds
2019-12-13 10:11:03,920 -- DEBUG -- Load section cloud in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,920 -- DEBUG -- cloud plugin - Create thread for OpenStack metadata
2019-12-13 10:11:03,921 -- DEBUG -- Plugin glances_cloud.py started in 0.001011 seconds
2019-12-13 10:11:03,923 -- DEBUG -- Starting new HTTP connection (1): 169.254.169.254:80
2019-12-13 10:11:03,923 -- DEBUG -- Load section alert in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:03,923 -- DEBUG -- Plugin glances_alert.py started in 0.002094 seconds
2019-12-13 10:11:03,924 -- DEBUG -- Load section ip in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:04,162 -- DEBUG -- Plugin glances_ip.py started in 0.238029 seconds
2019-12-13 10:11:04,162 -- DEBUG -- Load section uptime in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:04,162 -- DEBUG -- Plugin glances_uptime.py started in 0.000529 seconds
2019-12-13 10:11:04,162 -- DEBUG -- Load section irq in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:04,163 -- DEBUG -- Plugin glances_irq.py started in 0.000318 seconds
2019-12-13 10:11:04,163 -- WARNING -- Missing Python Lib (No module named 'pySMART'), HDD Smart plugin is disabled
2019-12-13 10:11:04,163 -- DEBUG -- Current user is not admin, HDD SMART plugin disabled.
2019-12-13 10:11:04,163 -- DEBUG -- Load section smart in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:04,163 -- DEBUG -- Plugin glances_smart.py started in 0.000411 seconds
2019-12-13 10:11:04,163 -- DEBUG -- Stats history activated for plugin diskio (items: ['read_bytes', 'write_bytes'])
2019-12-13 10:11:04,163 -- DEBUG -- Load section diskio in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:04,163 -- DEBUG -- Plugin glances_diskio.py started in 0.000299 seconds
2019-12-13 10:11:04,163 -- DEBUG -- Stats history activated for plugin load (items: ['min1', 'min5', 'min15'])
2019-12-13 10:11:04,164 -- DEBUG -- Load section load in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:04,164 -- DEBUG -- Load limit: load_careful = 0.7
2019-12-13 10:11:04,164 -- DEBUG -- Load limit: load_warning = 1.0
2019-12-13 10:11:04,164 -- DEBUG -- Load limit: load_critical = 5.0
2019-12-13 10:11:04,164 -- DEBUG -- Plugin glances_load.py started in 0.000416 seconds
2019-12-13 10:11:04,165 -- WARNING -- Sparklines module not found (No module named 'sparklines')
2019-12-13 10:11:04,165 -- WARNING -- Missing Python Lib (No module named 'cpuinfo'), Quicklook plugin will not display CPU info
2019-12-13 10:11:04,165 -- DEBUG -- Stats history activated for plugin quicklook (items: ['cpu', 'percpu', 'mem', 'swap'])
2019-12-13 10:11:04,165 -- DEBUG -- Load section quicklook in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:04,165 -- DEBUG -- Load limit: quicklook_cpu_careful = 50.0
2019-12-13 10:11:04,165 -- DEBUG -- Load limit: quicklook_cpu_warning = 70.0
2019-12-13 10:11:04,165 -- DEBUG -- Load limit: quicklook_cpu_critical = 90.0
2019-12-13 10:11:04,165 -- DEBUG -- Load limit: quicklook_mem_careful = 50.0
2019-12-13 10:11:04,165 -- DEBUG -- Load limit: quicklook_mem_warning = 70.0
2019-12-13 10:11:04,165 -- DEBUG -- Load limit: quicklook_mem_critical = 90.0
2019-12-13 10:11:04,165 -- DEBUG -- Load limit: quicklook_swap_careful = 50.0
2019-12-13 10:11:04,165 -- DEBUG -- Load limit: quicklook_swap_warning = 70.0
2019-12-13 10:11:04,165 -- DEBUG -- Load limit: quicklook_swap_critical = 90.0
2019-12-13 10:11:04,165 -- DEBUG -- Plugin glances_quicklook.py started in 0.001344 seconds
2019-12-13 10:11:04,165 -- DEBUG -- Load section folders in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:04,166 -- DEBUG -- Plugin glances_folders.py started in 0.000415 seconds
2019-12-13 10:11:04,166 -- DEBUG -- Load section psutilversion in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:04,166 -- DEBUG -- Plugin glances_psutilversion.py started in 0.000204 seconds
2019-12-13 10:11:04,284 -- DEBUG -- Stats history activated for plugin docker (items: ['cpu_percent'])
2019-12-13 10:11:04,284 -- DEBUG -- Load section docker in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:04,284 -- DEBUG -- Trying paths: ['/home/qubidt/.docker/config.json', '/home/qubidt/.dockercfg']
2019-12-13 10:11:04,284 -- DEBUG -- No config file found
2019-12-13 10:11:04,284 -- DEBUG -- Trying paths: ['/home/qubidt/.docker/config.json', '/home/qubidt/.dockercfg']
2019-12-13 10:11:04,284 -- DEBUG -- No config file found
2019-12-13 10:11:04,285 -- DEBUG -- Plugin glances_docker.py started in 0.118756 seconds
2019-12-13 10:11:04,285 -- DEBUG -- Stats history activated for plugin percpu (items: ['user', 'system'])
2019-12-13 10:11:04,285 -- DEBUG -- Load section percpu in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:04,285 -- DEBUG -- Load limit: percpu_user_careful = 50.0
2019-12-13 10:11:04,285 -- DEBUG -- Load limit: percpu_user_warning = 70.0
2019-12-13 10:11:04,285 -- DEBUG -- Load limit: percpu_user_critical = 90.0
2019-12-13 10:11:04,285 -- DEBUG -- Load limit: percpu_system_careful = 50.0
2019-12-13 10:11:04,285 -- DEBUG -- Load limit: percpu_system_warning = 70.0
2019-12-13 10:11:04,285 -- DEBUG -- Load limit: percpu_system_critical = 90.0
2019-12-13 10:11:04,285 -- DEBUG -- Plugin glances_percpu.py started in 0.000506 seconds
2019-12-13 10:11:04,285 -- DEBUG -- Stats history activated for plugin cpu (items: ['user', 'system'])
2019-12-13 10:11:04,285 -- DEBUG -- Load section cpu in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:04,285 -- DEBUG -- Load limit: cpu_user_careful = 50.0
2019-12-13 10:11:04,286 -- DEBUG -- Load limit: cpu_user_warning = 70.0
2019-12-13 10:11:04,286 -- DEBUG -- Load limit: cpu_user_critical = 90.0
2019-12-13 10:11:04,286 -- DEBUG -- Load limit: cpu_system_careful = 50.0
2019-12-13 10:11:04,286 -- DEBUG -- Load limit: cpu_system_warning = 70.0
2019-12-13 10:11:04,286 -- DEBUG -- Load limit: cpu_system_critical = 90.0
2019-12-13 10:11:04,286 -- DEBUG -- Load limit: cpu_steal_careful = 50.0
2019-12-13 10:11:04,286 -- DEBUG -- Load limit: cpu_steal_warning = 70.0
2019-12-13 10:11:04,286 -- DEBUG -- Load limit: cpu_steal_critical = 90.0
2019-12-13 10:11:04,286 -- DEBUG -- Load limit: cpu_iowait_careful = 3.333333333333333
2019-12-13 10:11:04,286 -- DEBUG -- Load limit: cpu_iowait_warning = 3.7499999999999996
2019-12-13 10:11:04,286 -- DEBUG -- Load limit: cpu_iowait_critical = 4.166666666666666
2019-12-13 10:11:04,286 -- DEBUG -- Load limit: cpu_ctx_switches_careful = 960000.0
2019-12-13 10:11:04,286 -- DEBUG -- Load limit: cpu_ctx_switches_warning = 1080000.0
2019-12-13 10:11:04,286 -- DEBUG -- Load limit: cpu_ctx_switches_critical = 1200000.0
2019-12-13 10:11:04,286 -- DEBUG -- Plugin glances_cpu.py started in 0.000773 seconds
2019-12-13 10:11:04,286 -- WARNING -- Missing Python Lib (No module named 'pymdstat'), Raid plugin is disabled
2019-12-13 10:11:04,286 -- DEBUG -- Load section raid in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:04,286 -- DEBUG -- Plugin glances_raid.py started in 0.000369 seconds
2019-12-13 10:11:04,287 -- DEBUG -- Load section core in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:04,287 -- DEBUG -- Plugin glances_core.py started in 0.000209 seconds
2019-12-13 10:11:04,287 -- DEBUG -- Load section now in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:04,287 -- DEBUG -- Plugin glances_now.py started in 0.000202 seconds
2019-12-13 10:11:04,287 -- DEBUG -- Stats history activated for plugin memswap (items: ['percent'])
2019-12-13 10:11:04,287 -- DEBUG -- Load section memswap in ['/home/qubidt/.config/glances/glances.conf', '/etc/glances/glances.conf']
2019-12-13 10:11:04,287 -- DEBUG -- Load limit: memswap_careful = 50.0
2019-12-13 10:11:04,287 -- DEBUG -- Load limit: memswap_warning = 70.0
2019-12-13 10:11:04,287 -- DEBUG -- Load limit: memswap_critical = 90.0
2019-12-13 10:11:04,287 -- DEBUG -- Plugin glances_memswap.py started in 0.000359 seconds
2019-12-13 10:11:04,287 -- DEBUG -- Active plugins list: ['connections', 'amps', 'fs', 'network', 'sensors', 'wifi', 'ports', 'gpu', 'help', 'system', 'processcount', 'processlist', 'mem', 'cloud', 'alert', 'ip', 'uptime', 'irq', 'diskio', 'load', 'quicklook', 'folders', 'psutilversion', 'docker', 'percpu', 'cpu', 'raid', 'core', 'now', 'memswap']
2019-12-13 10:11:04,287 -- DEBUG -- Active exports modules list: []
2019-12-13 10:11:04,289 -- DEBUG -- Server uses connections plugin
2019-12-13 10:11:06,925 -- DEBUG -- cloud plugin - Cannot connect to the OpenStack metadata API http://169.254.169.254/latest/meta-data/ami-id: HTTPConnectionPool(host='169.254.169.254', port=80): Max retries exceeded with url: /latest/meta-data/ami-id (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fbd157428e0>, 'Connection to 169.254.169.254 timed out. (connect timeout=3)'))
```

**Comment by nicolargo** [comment]
Hi @qubidt thanks for the issue report.

Concerning the PR, i think that that the first solution (correct the debug message) will solve this issue. When neither configuration file are provided, a default configuration is applied.

Can you test it and you are welcome to PR ;)

Nicolas

**Comment by qubidt** [comment]
Sorry! Looks like I'm a jerk. This was already fixed in abf64ffde31113f5f46ef286703ff061fc57395f:

```
diff --git a/glances/plugins/glances_plugin.py b/glances/plugins/glances_plugin.py
index 6a1c8606..23a76f80 100644
--- a/glances/plugins/glances_plugin.py
+++ b/glances/plugins/glances_plugin.py
@@ -86,10 +86,11 @@ class GlancesPlugin(object):
         self.stats_history = self.init_stats_history()
 
         # Init the limits (configuration keys) dictionnary
-        logger.debug('Load section {} in {}'.format(self.plugin_name,
-                                                    config.config_file_paths()))
         self._limits = dict()
-        self.load_limits(config=config)
+        if config is not None:
+            logger.debug('Load section {} in {}'.format(self.plugin_name,
+                                                        config.config_file_paths()))
+            self.load_limits(config=config)
 
         # Init the actions
         self.actions = GlancesActions(args=args)
```

After switching to `develop`, this call no longer throws an exception. The only issue is this has not been included in a release.

However, a bit down the line a different error occurs:

```
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 192, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/qubidt/github/glances/glances/__main__.py", line 29, in <module>
    glances.main()
  File "/home/qubidt/github/glances/glances/__init__.py", line 143, in main
    start(config=config, args=args)
  File "/home/qubidt/github/glances/glances/__init__.py", line 112, in start
    mode.serve_forever()
  File "/home/qubidt/github/glances/glances/client.py", line 238, in serve_forever
    if not self.login():
  File "/home/qubidt/github/glances/glances/client.py", line 162, in login
    if not self._login_glances():
  File "/home/qubidt/github/glances/glances/client.py", line 130, in _login_glances
    self.stats.set_plugins(json.loads(self.client.getAllPlugins()))
  File "/home/qubidt/github/glances/glances/stats_client.py", line 59, in set_plugins
    self._plugins[item] = plugin.Plugin(args=self.args)
  File "/home/qubidt/github/glances/glances/plugins/glances_processlist.py", line 120, in __init__
    if 'processlist' in config.as_dict() and 'sort_key' in config.as_dict()['processlist']:
AttributeError: 'NoneType' object has no attribute 'as_dict'
```

I'll push a PR for this.
